README.md

```diff
@@ -116,9 +115,37 @@ new Kafka({
 })
 ```
 
-#### <a name="setup-client-default-retry"></a> Default Retry
+### <a name="configuration-default-retry"></a> Default Retry
```

Comment:
I like the detailed explanation but can we move it to the bottom? I think the beginning of the readme should provide a quick setup and the most common use case, detailed information such as the custom logger or how the retry mechanism works could live further down. WDYT?